src/safe-upgrade-tool/developer.md

```diff
@@ -0,0 +1,150 @@
+---
+group: software-update-guide
+title: Developer information
+functional_areas:
+  - Upgrade
+---
+
+This topic contains information for developers who want to know more technical information about the Magento Safe Upgrade Tool (SUT). It is focused on developers who work closely with the Magento source code. You can use this knowledge to customize the SUT's components.
+
+## M-Ray integration
+
+M-Ray integration is an internal integration solution.
```

Comment:
I'm still not sure what M-Ray is. If it's internal, will the public have access to it?

Reply:
I'll chase that down, but I think it is an internal developed solution, but devs can access it

Reply:
Initially M-Ray along with SUT was supposed to be available only to Commerce customers. Has this changed and approved by product organization?

Reply:
@paliarush SUT will be limited to Commerce merchants as you pointed out there is no change on this. 
I think we should talk more about Magento API index instead of M-Ray, as M-Ray is an internal tool that won't be used by SUT users but the output (Magento API index) of it will. @paliarush what do you think?

Reply:
for the moment, changed it to Magento API index